packages/kit/test/apps/options/source/pages/host/index.svelte

```diff
@@ -1,7 +1,9 @@
 <script context="module">
+	import { base } from '$app/paths';
+
 	/** @type {import('@sveltejs/kit').Load} */
 	export async function load({ page, fetch }) {
-		const res = await fetch('/host.json');
+		const res = await fetch(`${base}/host.json`);
```

Comment:
Is this something you're going to want to do for every URL within the app or should we add a `base` html tag or something?

Reply:
It's same as we should do with links. 
Otherwise all `<a href="/any-path'` without `${base}` would point to wrong url. 

Without $base in url even we can intercept clicks and route such urls well, 
for crawlers, for open in new window that url would be wrong.

So every 'a' tag whould be like `<a href="{base}/path"`. _Not a big deal as Link component can be created_ 

`fetch` just same to be consistent.

Reply:
btw I havent knew about base tag. :-) so probably its much better. 

Would check tomorrow